### PR TITLE
Elasticsearch ACLs

### PR DESCRIPTION
**Pull request by ivan-savciuc**

### CODE REVIEW

File: aiven/provider.go [hunk 1]
```diff
@@ -28,6 +28,7 @@ func Provider() terraform.ResourceProvider {
 			"aiven_database":                     datasourceDatabase(),
 			"aiven_kafka_acl":                    datasourceKafkaACL(),
 			"aiven_kafka_topic":                  datasourceKafkaTopic(),
+			"aiven_elasticsearch_acls":           datasourceElasticsearchACLs(),
```
Comment (rikonen) [inline aiven/provider.go]: I think singular form would be better because everything else is in that form and ACL naturally contains multiple entries.

Comment (ivan-savciuc) [reply aiven/provider.go]: indeed, I will change that, thanks

File: README.md [hunk 2]
```diff
@@ -391,6 +391,63 @@ topics (admin, read, readwrite, write).
 Aiven ID format when importing existing resource: `<project_name>/<service_name>/<acl_id>`.
 The ACL ID is not directly visible in the Aiven web console.
 
+### Resource Elasticsearch ACL
+
+```
+resource "aiven_elasticsearch_acls" "es-acls" {
```
Comment (rikonen) [inline README.md]: `aiven_elasticsearch_acls` => `aiven_elasticsearch_acl`

File: examples/elasticsearch/elasticsearch_acls.tf [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Elasticsearch ACLs
+resource "aiven_elasticsearch_acls" "es-acls" {
```
Comment (rikonen) [inline examples/elasticsearch/elasticsearch_acls.tf]: `aiven_elasticsearch_acls` => `aiven_elasticsearch_acl`

Comment (ivan-savciuc) [reply examples/elasticsearch/elasticsearch_acls.tf]: done

File: README.md (hunk 2 above)
Comment (ivan-savciuc) [reply README.md]: done

**Pull request merged**